src/fides/api/api/v1/endpoints/storage_endpoints.py

```diff
@@ -104,7 +104,7 @@ def upload_data(
     logger.info("Starting storage upload for request id: {}", request_id)
     try:
         data_location: str = upload(
-            db, request_id=request_id, data=data, storage_key=storage_key
+            db, privacy_request=privacy_request, data=data, storage_key=storage_key
```

Comment:
The DSR HTML package needs a bit more info from the privacy request than just the ID so I updated a few functions to forward the entire privacy request object instead of just the ID.